### DISCUSSION TITLE
为什么需要开代理才能打开首页，不开代理打不开？

### DISCUSSION THREAD

**Discussion by OneSentry** [spencerwooo/onedrive-vercel-index#869] [Q&A] [answered]
什么都搭建好了，也运营起来了，为什么需要开代理才能打开首页，不开代理打不开，我打开别人的为什么不需要代理，

这是我自己搭建的地址
https://pla-onedrive-vercel-index-gaw9.vercel.app/zh-CN/

请大神指点一下

**Comment by spencerwooo** [comment, marked as answer]
是域名 `vercel.app` 的问题，被屏蔽了。

**Comment by Gavin-Meskin-Chen** [comment]
建议买个域名，现在新人第一年只要1块钱。或者还有免费域名。然后去vercel换上就行了。
